docs/releases/3.0.txt

```diff
@@ -64,7 +64,8 @@ Minor features
 :mod:`django.contrib.gis`
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* Added support on MySQL for spatial lookup functions operating on real
```

Comment:
Allowed MySQL spatial lookup functions to operate on real geometries. Previous support was limited to bounding boxes.